quickstarts/microsoft.hybridcompute/machine-windows-configuration/README.md

```diff
@@ -0,0 +1,89 @@
+# Windows VM with Azure secure baseline
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/CredScanResult.svg)
+
+![Bicep Version](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.compute/vm-windows-baseline/BicepVersion.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.compute%2Fvm-windows-baseline%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.compute%2Fvm-windows-baseline%2Fazuredeploy.json)
+
+This template allows you to deploy a Windows VM with the Azure secure baseline applied. For details about the settings in the baseline,
+review the reference documentation.
+
+[Windows secure baseline](https://docs.microsoft.com/en-us/azure/governance/policy/samples/guest-configuration-baseline-windows)
+
+General information about how configurations are assigned to machines in Azure is available in documentation.
+
+[Understand the guest configuration feature of Azure Policy](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/guest-configuration)
+
+A detailed how to document about assigning configurations to machines, and how to customize configurations from ARM,
+is also available.
+
+## Required prerequisites
+
+Before testing this quickstart template, you must create a Windows machine
+(virtual or physical) outside of Azure and install the Arc agent
+to project it into Azure.
+
+The simplest approach to building this prereq is to follow the docs page
```

Comment:
I know we briefly chatted about this on IM, but it might also be worth linking to the [Jumpstart template](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/azure/azure_arm_template_win/) for folks who want to quickly spin up a correctly-configured Azure VM that mimics an on-prem VM (has the guest agent disabled and blocks Azure IMDS).

Reply:
added